latex/ledger-spec.tex

```diff
@@ -429,6 +435,94 @@ \section{Addresses}
 
 \clearpage
 
+\section{Protocol Parameters}
+\label{sec:protocol-parameters}
+
+The $\PParams$ is
+an abstract type that will represent an environment variable that contains
+values on which the functionality of the blockchain protocol depends, such
+as the fees transactions are obligated to pay to be processed.
+
+TODO - move explanations of indididual parameters explanation here.
```

Comment:
would it make sense to use some latex \todo or similar, in order to make this better visible